languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37515,8 +37690,8 @@ though, and has since been slightly modified)-->
                 <token regexp="yes">tâches?</token>
             </pattern>
             <message>« Assigner (quelqu’un) à une tâche » peut être considéré comme un anglicisme.</message>
-            <suggestion>affecteur</suggestion>
-            <example correction="affecteur"><marker>assigner</marker> à une tâche</example>
+            <suggestion>affecter</suggestion>
```

Comment:
Well spotted, but this rule has more problems: it takes any form of "assigner", but it always only gives "affecter" as a suggestion. Can be resolved with

```xml
<suggestion><match no="1" postag="V.*" postag_regexp="yes">affecter</match></suggestion>
```

to give "affecter" the same form that "assigner" has